doc/installation.md

```diff
@@ -50,9 +50,15 @@ You need to create a database and database-user to use with Tracks. For this, yo
 
 Tracks is built upon a number of Ruby libraries (known as ‘gems’). The Bundler tool makes it easy to install all the gems that Tracks needs, and ensures that they are all the correct versions.
 
-1. Edit the file `Gemfile` in the Tracks root directory. You may comment out any database drivers you will not be using. You may also comment out `therubyracer` if you are installing Tracks on Windows or Mac OS X, or if you have another JavaScript runtime such as Node.js installed.
-2. Run the command `bundle install --without development test` in the directory that you unzipped your Tracks download to.
-3. Wait for Bundler to finish installing the necessary gems that Tracks needs. This can take some time depending on the speed of your internet connection and the speed of the system you’re installing Tracks on.
+1. If you are installing Tracks on Windows or Mac OS X, or if you have another JavaScript runtime such as Node.js installed, edit the file `Gemfile` in the Tracks root directory, and  comment out `therubyracer`.
+1. In the directory that you unzipped your Tracks download to, install all the appropriate gems by running:
+
+        # For SQLite users:
+        bundle install --without development test mysql
+        # For MySQL users:
+        bundle install --without development test sqlite
+
+1. Wait for Bundler to finish installing the necessary gems that Tracks needs. This can take some time depending on the speed of your internet connection and the speed of the system you’re installing Tracks on.
```

Comment:
Please keep the actual numbering -- it is a little tedious when making changes, but it keeps the file more readable as plaintext for users who are opening the file from within their downloaded copies of Tracks.